app/etc/di.xml

```diff
@@ -407,11 +407,16 @@
     </type>
     <type name="Magento\Framework\Interception\Config\Config">
         <arguments>
-            <argument name="cache" xsi:type="object">Magento\Framework\App\Cache\Type\Config</argument>
```

Comment:
I don't think that we need to remove it. Try to find how to mark them as deprecated in Devdocs.